# gripper_params.yaml
Gripper Parameter:
  use_brk:
    type: bool
    description: "Use brake"
  grp_pos_margin:
    type: double
    description: "Grip position margin in mm"
  grp_prepos_delta:
    type: double
    description: "Grip position delta in mm"
  zero_pos_ofs:
    type: double
    description: "Zero position offset in mm"
  grp_prehold_time:
    type: int
    description: "Grip prehold time in ms"
  wp_release_delta:
    type: double
    description: "Workpiece release delta in mm"
  wp_lost_distance:
    type: double
    description: "Max. distance after workpiece lost in mm"

Control Parameter:
  move_gripper:
    type: double
    description: "Moving the gripper with parameter in mm"
  move_gripper_velocity:
    type: double
    description: "Changing the velocity for move_Gripper in mm/s"
  grip_direction:
    type: bool
    description: "Gripdirection for Parameter gripping"
  grip:
    type: bool
    description: "Grip with parameter"
  grip_force:
    type: double
    description: "Gripforce in %"
  release_workpiece:
    type: bool
    description: "Release Workpiece"
